maphacks/z2a4.mhk

```diff
@@ -0,0 +1,23 @@
+// Handsoap Bottles in the perfume store
+
+// Left
+sprite 2262 angoff 1536
```

Comment:
I'd add comment to every line with voxel's name. Take a look at Duke Nukem voxel pack.

Reply:
Added descriptions to each line